.github/workflows/downgraded_release.yaml

```
@@ -35,6 +35,22 @@ jobs:
             # downgrade to PHP 7.2
             -   run: rector-local/vendor/bin/rector process bin src packages vendor --config build/rector-downgrade-php-72.php --ansi
 
+            # verify PHP 7.2 compat
+            -
+                uses: "shivammathur/setup-php@v2"
+                with:
+                    php-version: 7.2
+                    coverage: none
+
+            -   run: vendor/bin/parallel-lint src packages vendor
```

should this be a different path

```suggestion
            -   run: bin/parallel-lint src packages vendor
```

like ECS is used in line 55?